packages/blockchain-link-utils/src/solana.ts

```diff
@@ -0,0 +1,45 @@
+import { ParsedTransactionWithMeta } from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+export function extractAccountBalanceDiff(
```

Comment:
First step in parsing a tx, is getting Solana effects on accounts that were in the transaction, from this effects we later parse the other tx properties.

Reply:
honestly, I wouldn't mind having these comments in comments in code. great for future generations that will be exploring this realm

Reply:
in [chore(blockchain-link): add explanatory comments](https://github.com/trezor/trezor-suite/pull/9498/commits/87b52c77fe7e21551bb126418194c83ce4d07746)